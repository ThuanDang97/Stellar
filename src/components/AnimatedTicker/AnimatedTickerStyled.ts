import styled, { keyframes } from 'styled-components'

// Components
import {
  ButtonLinkStyled,
  TextStyled,
  ViewStyled,
} from '@components/styled-components'

// Props type
import { AnimatedWrapperProps } from '@self-types/components/Animation.props'

// Utils
import { pxToRem } from '@utils/theme'

const slide = keyframes`
  to {
    transform: translateX(-100%)
  }
`

const fadeIn = keyframes`
  from {
    opacity: 0
  }
  to {
    opacity: 1
}
`

const fadeOut = keyframes`
  from {
    opacity: 1
  }
  to {
    opacity: 0
  }
`

export const ContentWrapper = styled(ViewStyled)<AnimatedWrapperProps>`
  display: flex;
  width: 100%;

  & > ${ViewStyled} {
    flex: 0 0 100%;

    &:nth-child(1) {
      animation: ${fadeOut} 1s linear both
          ${({ delay }) => `${(delay ?? 0) + 11}s`},
        ${slide} 0s linear forwards ${({ delay }) => `${(delay ?? 0) + 12}s`};
    }

    &:nth-child(2) {
      animation: ${slide} 0s linear both
          ${({ delay }) => `${(delay ?? 0) + 12}s`},
        ${fadeIn} 1s ease-out both ${({ delay }) => `${(delay ?? 0) + 13}s`};

      & > ${TextStyled} {
        width: min(90%, ${pxToRem(600)});
      }

      & > ${ButtonLinkStyled} {
        animation: ${fadeIn} 1s ease both
          ${({ delay }) => `${(delay ?? 0) + 18}s`};
      }
    }
  }
`

export const TitleWrapperStyled = styled(ViewStyled)<AnimatedWrapperProps>`
  display: flex;
  width: min(90%, ${pxToRem(550)});
  margin-bottom: ${({ theme }) => pxToRem(theme.metrics.dimensions.lg)};

  overflow: hidden;

  ${TextStyled} {
    flex: 0 0 100%;

    &:nth-child(1) {
      animation: ${fadeOut} 0.5s linear both
          ${({ delay }) => `${(delay ?? 0) + 4.8}s`},
        ${slide} 0s linear forwards ${({ delay }) => `${(delay ?? 0) + 5.2}s`};
    }

    &:nth-child(2) {
      animation: ${slide} 0s linear both
          ${({ delay }) => `${(delay ?? 0) + 4.5}s`},
        ${fadeIn} 1s ease-out both ${({ delay }) => `${(delay ?? 0) + 4.8}s`};
    }
  }
`
