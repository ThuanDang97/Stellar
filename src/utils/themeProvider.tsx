import { ThemeProvider } from 'styled-components'
import { render } from '@testing-library/react'
import { theme } from '@themes/index'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>)
}
