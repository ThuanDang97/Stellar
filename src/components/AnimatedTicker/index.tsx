import { memo } from 'react'

// Components
import { ViewStyled } from '@components/styled-components'
import Ticker from '@components/Ticker'
import LinkComponent from '@components/Link'

// Themes
import { useTheme } from 'styled-components'

// Props type
import { AnimationComponentProps } from '@self-types/components/Animation.props'

// Constants
import { DEFAULT_HEADER_URL } from '@constants/routes'
import { ANIMATION_TICKER_TITLE } from '@constants/animation'

// Styles
import { ContentWrapper, TitleWrapperStyled } from './AnimatedTickerStyled'
import { TitleLandingPageStyled } from '../../../styles/pages/LandingPageStyled'

const AnimatedTicker = ({ delay = 0 }: AnimationComponentProps) => {
  const theme = useTheme()

  return (
    <ContentWrapper delay={delay}>
      {/* Ticker animation */}
      <ViewStyled display="flex" flexDirection="column">
        <TitleWrapperStyled delay={delay}>
          {ANIMATION_TICKER_TITLE.map((title) => (
            <TitleLandingPageStyled
              key={title}
              fontSize={theme.typography.fontSize.xlg}
              fontWeight={theme.typography.fontWeight.bold}
              color={theme.colors.white}
              letterSpacing={theme.typography.letterSpacing.xxl}
              lineHeight={theme.typography.lineHeight.lg}
              as="h1"
            >
              {title}
            </TitleLandingPageStyled>
          ))}
        </TitleWrapperStyled>

        <Ticker
          size={70}
          firstNumber={679}
          secondNumber={752}
          duration={2}
          delay={delay}
        />
      </ViewStyled>

      {/* Switch Off animation */}
      <ViewStyled display="flex" flexDirection="column">
        <TitleLandingPageStyled
          fontSize={theme.typography.fontSize.xlg}
          fontWeight={theme.typography.fontWeight.bold}
          color={theme.colors.white}
          letterSpacing={theme.typography.letterSpacing.xxl}
          lineHeight={theme.typography.lineHeight.lg}
          as="h1"
        >
          Switch off the gravity holding down your score.
        </TitleLandingPageStyled>

        <LinkComponent
          href={DEFAULT_HEADER_URL.SIGN_UP.URL}
          linkTypes="continueLink"
          text="Continue"
          width={315}
          mTop={theme.metrics.dimensions.xxl}
        />
      </ViewStyled>
    </ContentWrapper>
  )
}

export default memo(AnimatedTicker)
