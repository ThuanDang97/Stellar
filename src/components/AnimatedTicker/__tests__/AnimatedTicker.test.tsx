import React from 'react'

// Utils
import { renderWithTheme } from '@utils/themeProvider'

// Components
import AnimatedTicker from '..'

describe('AnimatedTicker Component', () => {
  it('renders AnimatedTicker unchanged', () => {
    const { container } = renderWithTheme(<AnimatedTicker />)

    expect(container).toMatchSnapshot()
  })
})
