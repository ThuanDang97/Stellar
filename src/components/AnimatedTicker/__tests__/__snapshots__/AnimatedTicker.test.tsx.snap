// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedTicker Component renders AnimatedTicker unchanged 1`] = `
<div>
  <div
    class="ViewStyled-sc-rqe5sa-0 AnimatedTickerStyled__ContentWrapper-sc-nln3s6-0 fTsvHV jVekgM"
  >
    <div
      class="ViewStyled-sc-rqe5sa-0 kBewKp"
      display="flex"
    >
      <div
        class="ViewStyled-sc-rqe5sa-0 AnimatedTickerStyled__TitleWrapperStyled-sc-nln3s6-1 fTsvHV giwhCe"
      >
        <h1
          class="TextStyled-sc-9xia41-0 LandingPageStyled__TitleLandingPageStyled-sc-19vluzy-2 qecIO ekfgDB"
          color="#FFF"
          font-size="36"
          font-weight="700"
          letter-spacing="0.9"
        >
          Building your credit score and credit profile over time.
        </h1>
        <h1
          class="TextStyled-sc-9xia41-0 LandingPageStyled__TitleLandingPageStyled-sc-19vluzy-2 qecIO ekfgDB"
          color="#FFF"
          font-size="36"
          font-weight="700"
          letter-spacing="0.9"
        >
          The more bills you add, the more your score can improve.
        </h1>
      </div>
      <div
        class="ViewStyled-sc-rqe5sa-0 TickerStyled__TickerWrapperStyled-sc-x8b1pu-0 eQdhWe inONBg"
        color="#F22E3E"
        data-testid="digit-container"
        height="0"
      >
        <div
          class="TickerStyled__TickerColumn-sc-x8b1pu-1 TickerStyled__TickerColumnOne-sc-x8b1pu-2 ejdlmi duqavd"
        >
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
        </div>
        <div
          class="TickerStyled__TickerColumn-sc-x8b1pu-1 TickerStyled__TickerColumnTwo-sc-x8b1pu-3 ejdlmi JKGqo"
        >
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
        </div>
        <div
          class="TickerStyled__TickerColumn-sc-x8b1pu-1 TickerStyled__TickerColumnThree-sc-x8b1pu-4 ejdlmi lpsVtF"
        >
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            0
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            1
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            2
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            3
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            4
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            5
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            6
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            7
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            8
          </h2>
          <h2
            class="TickerStyled__TickerDigit-sc-x8b1pu-5 NgIke"
            font-size="70"
          >
            9
          </h2>
        </div>
      </div>
      <div
        class="TickerStyled__TickerSimulator-sc-x8b1pu-6 eiGKYT"
        font-size="70"
      >
        0
      </div>
    </div>
    <div
      class="ViewStyled-sc-rqe5sa-0 kBewKp"
      display="flex"
    >
      <h1
        class="TextStyled-sc-9xia41-0 LandingPageStyled__TitleLandingPageStyled-sc-19vluzy-2 qecIO ekfgDB"
        color="#FFF"
        font-size="36"
        font-weight="700"
        letter-spacing="0.9"
      >
        Switch off the gravity holding down your score.
      </h1>
      <a
        class="LinkStyled__ButtonLinkStyled-sc-626068-1 ilolYJ"
        data-testid="link"
        href="/sign-up"
        width="315"
      >
        Continue
      </a>
    </div>
  </div>
</div>
`;
